superset-frontend/src/explore/exploreUtils.js

```diff
@@ -309,19 +310,28 @@ export const useDebouncedEffect = (effect, delay) => {
   }, [callback, delay]);
 };
 
-export const getSimpleSQLExpression = (
-  subject,
-  operator,
-  comparator,
-  isMulti,
-) => {
+export const getSimpleSQLExpression = (subject, operator, comparator) => {
+  const isMulti = MULTI_OPERATORS.has(operator);
   let expression = subject ?? '';
   if (subject && operator) {
     expression += ` ${operator}`;
-    if (comparator) {
-      expression += ` ${isMulti ? "('" : ''}${comparator}${
-        isMulti ? "')" : ''
-      }`;
+    const firstValue =
+      isMulti && Array.isArray(comparator) ? comparator[0] : comparator;
+    const comparatorArray =
+      comparator === undefined ||
+      comparator === null ||
```

Comment:
What do you think about setting `comparatorArray` as an empty array if `comparator` is null or undefined? Then we wouldn't need `comparatorArray || []` in line 330

Reply:
I wanted to squeeze this in a ternary, but I agree, this probably makes it more readable.